src/commands/init_mission.rs

```diff
@@ -0,0 +1,15 @@
+use anyhow::{Result, anyhow};
+
+use crate::{config::E4EDMConfig, dataset::build_mission_metadata};
+
+use super::InitMissionArgs;
+
+pub(crate) fn init_mission(args: &InitMissionArgs, config: &mut E4EDMConfig) -> Result<()> {
+    let active_dataset_name = config.active_dataset.clone()
+        .ok_or(anyhow!("Dataset not active!"))?;
```

Comment:
See my earlier comment about the `let else` block - that's probably the best option in terms of readability and brevity.

Reply:
Good find! Entirely removes the need for `ok_or()` and the `?` operator. Change is included in my most recent commit.